### PR TITLE
Ep25: RustFest w/ Katharina & Florian

### PR DESCRIPTION
**Pull request by bstrie**
The Florian interview was a bit freeform so timestamps for discrete topics didn't really make sense.

### CODE REVIEW

File: _episodes/025-katharina-florian.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: "RustFest 2019 Interview Series: Burnout in Open Source Software; The Rust Roadmap"
+date: 2015-07-30T22:00:00Z
```
Comment (jonhoo) [inline _episodes/025-katharina-florian.md:3]: Don't forget to update this :sweat_smile:

File: _episodes/025-katharina-florian.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+ - [@01:54] - How common is burnout in software?
+ - [@03:24] - How does burnout manifest in volunteer endeavors like open source software?
+ - [@08:10] - How does rotation of responsibilities alleviate burnout?
+ - [@13:41] - What communities succeed at combating burnout?
+ - [@16:44] - Final thoughts on burnout and governance
+
+#### [@19:50] Part 2: The Rust Roadmap w/ Florian Gilcher
+
+<!--
```
Comment (jonhoo) [inline _episodes/025-katharina-florian.md:42]: I wonder — are there useful links in what you talked to Florian about? Like a link to the roadmap possibly?

Comment (bstrie) [reply _episodes/025-katharina-florian.md:42]: I could link to the 2019 roadmap, I suppose I could link to the 2020 roadmap as well although at the time it didn't exist, so we were mostly just talking about it in abstract, would that be weird?

Comment (jonhoo) [reply _episodes/025-katharina-florian.md:42]: I think linking to both would be a good idea! It's something the listeners are likely to care about after listening to it :)

Comment (jonhoo) [reply _episodes/025-katharina-florian.md:42]: Gentle nudge on this

**Pull request merged**